test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,36 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHasAnyNull() throws Exception {
```

Comment:
The test itself is correctly written,
But, is `elementsHasAnyNull` the name of the method you are testing?
- Remember that the first part of the name is the name of the method you are testing.
- Please also be aware of the 3 part test name format that may be useful when you write tests for your project.